.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: release
+on:
+  push:
+    tags:
+      - '[0-9]*'
```

Comment:
what about point releases? `12.1` would not match this regex.

Reply:
This is not a regexp, it's a [glob](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags). Thus this just means "tag must start with a number".

Reply:
I added a comment about this.